tests/componenttests/server/common/MessageBuilders.cpp

```diff
@@ -54,6 +54,36 @@ ::firebolt::rialto::CreateSessionRequest createCreateSessionRequest(const VideoR
     return request;
 }
 
+::rialto::SetStateRequest createSetStateRequest(::rialto::SessionServerState value)
+{
+    ::rialto::SetStateRequest request;
+    request.set_sessionserverstate(value);
+    return request;
+}
+
+::rialto::SetLogLevelsRequest createSetLogLevelsRequest()
+{
+    const ::google::protobuf::uint32 kLogLevel = 2;
+    ::rialto::SetLogLevelsRequest request;
+    auto levels = request.mutable_loglevels();
+
+    levels->set_defaultloglevels(kLogLevel);
```

Comment:
Is setting the log levels to different values a better test? It would help determine if theres a mix up in parameters and values.

Reply:
> > Is setting the log levels to different values a better test? It would help determine if theres a mix up in parameters and values.
> 
> This does seem like a good suggestion.
> 
> But there doesn't seem to be a way to obtain the current loglevels from RialtoApplicationSessionServer (so that the setting can actually be verified) ? Should we extend the proto messages to include new methods (GetLogLevelsRequest and GetLogLevelsResponse)?

The server runs within a different thread in the test application therefore a solution is to make calls to getLogLevels().  This "backdoor" solution works, I'll submit it soon